app/presenters/indexable_presenter.rb

```diff
@@ -0,0 +1,25 @@
+class IndexablePresenter
+
+  def initialize(policy)
+    @policy = policy
+  end
+
+  def indexable_attributes
+    {
+      title: policy.name,
```

Comment:
I think this set of properties is fine to start with, but it would be good to consider whether the difference between policies and programmes should be made available to the search index (as an additional field - I think that might be useful information for ranking).  Maybe that should be considered as a subsequent PR - I'm not sure how that's being modelled at the moment.

I assume we'll need to record whether this is a "current" policy or not, too, but again that can be a later PR.